Woof/Woof/View/ReusableViews/LabelWithTextAndImage.swift

```diff
@@ -0,0 +1,25 @@
+import SwiftUI
+
+/// A view displaying an icon and the portion of text.
+struct LabelWithTextAndImage: View {
```

Comment:
IIRC, we had an agreement to put the `View` suffix:
```suggestion
struct LabelWithTextAndImageView: View {
```

Reply:
Also you can make it simple:
```suggestion
struct ImageWithLabelView: View {
```

Reply:
Thanks for the suggestion! 
After discussion with the team, we decided that we don't need this reusable element.
So the file `LabelWithTextAndImage.swift` are deleted.